styleguide/derek/pattern-components/service-level-teaser/_service-level-teaser.scss

```diff
@@ -9,10 +9,15 @@
   position: relative;
   text-decoration: none;
   width: 100%;
+}
 
-  &:hover {
-    background-color: $gray-lightest;
-    text-decoration: none;
+// only add a hover class if a link
+a {
+  &.serviceLevelTeaser {
```

Comment:
We should be using a separate class for links, instead of attaching to an anchor.  Adding a separate `.serviceLevelTeaser-link` would make sense.